### PR TITLE
fix issue with bulk insert when database table not exists.

### PR DESCRIPTION
**Pull request by Wu-Jing**
https://trello.com/c/Jrb59w6W

By the way I left some comments for you on slack. 

Signed-off-by: Jing <wujing1921@gmail.com>

### CODE REVIEW

File: executors/embedded-executor/src/main/scala/yumi/execution/RunEmbeddedExecutor.scala [hunk 1]
```diff
@@ -17,47 +17,48 @@ object RunEmbeddedExecutor extends App with Logging {
...
+        pipelineDefinition =
+          """
+            |{
+            |  "name": "test pipeline",
+            |  "activities": [
+            |     {
+            |       "name": "load csv file",
+            |       "typeUri": "yumi:loadTable",
+            |       "parameters": {
+            |          "path": "/Users/Shared/yumi/yumi-test.csv",
```
Comment (Wu-Jing) [inline executors/embedded-executor/src/main/scala/yumi/execution/RunEmbeddedExecutor.scala]: this line is changed (the format is off)

File: executors/embedded-executor/src/main/scala/yumi/execution/RunEmbeddedExecutor.scala [hunk 2]
```diff
@@ -17,47 +17,48 @@ object RunEmbeddedExecutor extends App with Logging {
...
+            |       "parameters": {
+            |          "table": "FamousPeople"
+            |       }
+            |     },
+            |     {
+            |       "name": "write to mysql",
+            |       "typeUri": "yumi:mySqlWriter",
+            |       "parameters": {
+            |          "table": "FamousPeople",
+            |          "url": "jdbc:mysql://yumi-dev.ckpgpq1o4ip9.us-east-1.rds.amazonaws.com:3306/yumi?user=yumi&password=abc123456",
```
Comment (Wu-Jing) [inline executors/embedded-executor/src/main/scala/yumi/execution/RunEmbeddedExecutor.scala]: this line is changed

File: pipeline/src/main/scala/yumi/pipeline/YumiContextFactory.scala [hunk 3]
```diff
@@ -20,12 +20,17 @@ class YumiContextFactory {
   private[this] def createSparkConf(config: Config): SparkConf = {
     val sparkConf = new SparkConf()
 
-    if(config.hasPath("yumi.spark.master")) {
+    if (config.hasPath("yumi.spark.master")) {
       sparkConf.setMaster(config.getString("yumi.spark.master"))
     }
+    else {
+      sparkConf.setMaster("local[*]")
+    }
```
Comment (Wu-Jing) [inline pipeline/src/main/scala/yumi/pipeline/YumiContextFactory.scala]: I think I'm missing a conf file. (see slack for comments) IIRC you created a conf file with these configs. For now I set it to default if no conf is provided.

File: project/Dependencies.scala [hunk 4]
```diff
@@ -34,9 +34,9 @@ object Dependencies {
   )
 
   lazy val sparkLibs = Seq(
-    "org.apache.spark" %% "spark-core" % sparkVersion % Provided excludeAll(excludeNettyIo, excludeQQ),
```
Comment (Wu-Jing) [inline project/Dependencies.scala]: What do you think about remove the "provided"? We don't want to download spark if user already have it. but if they don't they don't need to manually install it, plus they don't need to worry about which version of spark to download.

File: pipeline/src/main/scala/yumi/pipeline/YumiContextFactory.scala (hunk 3 above)
Comment (coderdiaries) [inline pipeline/src/main/scala/yumi/pipeline/YumiContextFactory.scala]: conf should be there, lets remove else part.

File: project/Dependencies.scala (hunk 4 above)
Comment (coderdiaries) [inline project/Dependencies.scala]: Spark has to be there for yumi to run. It should be `Provided`.

Comment (Wu-Jing) [inline project/Dependencies.scala]: Done

File: pipeline/src/main/scala/yumi/pipeline/YumiContextFactory.scala (hunk 3 above)
Comment (Wu-Jing) [inline pipeline/src/main/scala/yumi/pipeline/YumiContextFactory.scala]: done

**Comment by Wu-Jing** [comment]
@coderdiaries  I will close this pull request. I saw your commit on the feature branch, much better solution....

**Pull request closed without merging**